minpy/core.py

```diff
@@ -15,16 +15,19 @@
 _logger = log.get_logger(__name__)
 
 
-def grad_and_loss(func, argnum=0):
+def grad_and_loss(func, argnum=0, method=False):
```

Comment:
Maybe we can improve in the future, but now, self, as the first arg, will be converted into MinPy array. Like this:
`arrays = tuple(array.Value.wrap(a) for a in args)`
I want to avoid this issue without a bigger change.